lib/travis/api/v3/queries/request.rb

```diff
@@ -22,7 +22,11 @@ def schedule(repository, user)
         config:     config || {}
       }
 
-      perform_async(:build_request, type: 'api'.freeze, credentials: { token: token }, payload: JSON.dump(payload))
```

Comment:
I believe the queue name is always `build_requests`. `Query.sidekiq_queue()` has some code that pluralizes the input.